app/services/idv/vendor_validator.rb

```diff
@@ -25,5 +25,22 @@ def idv_agent
         vendor: (idv_session.vendor || idv_vendor.pick)
       )
     end
+
+    def result
+      @_result ||= try_submit
+    end
+
+    def try_agent_action
+      yield
+    rescue => err
+      err_msg = err.to_s
```

Comment:
I agree, we should handle them consistently with how we handle other exceptions. I assumed we were monitoring logs for alert generation for other patterns like this.

would love @jgrevich @amoose @timothy-spencer to comment